### PR TITLE
Fix: Onboarding updates to user hamburger menu

### LINKED ISSUE #1675: Onboarding: Updates to user hamburger menu
## Description 

When connecting as a new user without a profile created, the top right hamburger menu shows 'Manage account'. When selected, it does nothing.

image

## PR scope:

1. The scope of this PR is to update the user menu to the following design to show 'Create profile' instead of 'Manage account' only in this state where the user does not have a profile created.

Once a user creates a profile, the text should switch to the 'Manage account'.

2. Also, the user name (above the address next to the avatar) should be the user's truncated wallet address and updates once the user completes the profile creation flow. Currently it's showing three ellipsis. 

3. In the disconnected state, the floating HR at the bottom should be hidden until the light and dark mode features are put back into beta.

image

## Figma: 

Link to design: https://www.figma.com/file/l1dOM5qiQYwF0ElvKDqqjg/Design-System---Colony-v3?type=design&node-id=1876-9135&mode=design&t=4yuQP2mYSFlsEmh9-4

### PR DESCRIPTION
**Pull request by iamsamgibbs**
## Description

In the user hamburger menu, if the user does not have a profile show "Create profile" instead of "Manage account". Also, in place of their username, show their truncated wallet address.

In the wallet disconnected state, removed the border bottom dividing line from the bottom of the user hamburger menu.

**Changes** 🏗

* "Manage account" button now shows "Create profile" text and links to create profile route when the user has no profile.
* When the user has no profile, the username now shows their truncated wallet address instead of an ellipsis.
* The border bottom dividing line now only shows when a wallet is connected (thereby removing it when disconnected)
* userMenu.manegeTitle typo corrected

User hamburger menu when user has no profile
Screenshot 2024-01-16 at 13 30 52

User hamburger menu when user has already created a profile
Screenshot 2024-01-16 at 13 32 54

Wallet disconnected state (dividing line removed)
Screenshot 2024-01-16 at 13 33 15

Resolves #1675

### CODE REVIEW

**Review by melyndav — APPROVED** [review]
Ohhh this is really nice work @iamsamgibbs! All issues have been resolved and working as expected. Thank you for picking this issue up. 👍

**Review by rdig — DISMISSED** [review]
Nice work!

**Review by melyndav — APPROVED** [review]

**Review by rdig — APPROVED** [review]
Reapproved!

File: src/components/common/Extensions/UserNavigation/partials/WalletConnectedTopMenu/WalletConnectedTopMenu.tsx [hunk 1]
```diff
@@ -32,13 +37,27 @@ const WalletConnectedTopMenu: FC<UserAvatarDetailsProps> = ({
...
+          className="navigation-link hover:bg-gray-50 rounded -ml-4 w-[calc(100%+2rem)]"
+        >
+          <Icon name="user-circle-gear" appearance={{ size: iconSize }} />
+          <p className="ml-2">
+            {formatMessage({ id: 'userMenu.manageTitle' })}
+          </p>
+        </Link>
+      ) : (
+        <Link
+          to={`${CREATE_PROFILE_ROUTE}`}
```
Comment (ArmandoGraterol) [inline src/components/common/Extensions/UserNavigation/partials/WalletConnectedTopMenu/WalletConnectedTopMenu.tsx:52]: We don't need the template string here, we can write it like this: `to={CREATE_PROFILE_ROUTE}`

**Review by ArmandoGraterol — APPROVED** [review]
Nice work Sam! Just noticed a small detail but other than that I'm good with this

File: src/components/common/Extensions/UserNavigation/partials/WalletConnectedTopMenu/WalletConnectedTopMenu.tsx (hunk 1 above)
Comment (iamsamgibbs) [reply src/components/common/Extensions/UserNavigation/partials/WalletConnectedTopMenu/WalletConnectedTopMenu.tsx]: Totally right, removed this! :)

**Review by ArmandoGraterol — APPROVED** [review]

**Review by rdig — APPROVED** [review]
Re-approved, yet again, so you can merge

**Review by rdig — DISMISSED** [review]

**Review by ArmandoGraterol — APPROVED** [review]

**Review by melyndav — APPROVED** [review]
Re-approving for merge @iamsamgibbs 👍

**Review by rdig — APPROVED** [review]

**Pull request merged**